### PR TITLE
Convert post on OTel governance to Markdown and publish it

### LINKED ISSUE #5033: Blog post: Behind the scenes of the OpenTelemetry Governance Committee
I'd like to propose the following blog post: Behind the scenes of the OpenTelemetry Governance Committee

https://docs.google.com/document/d/1Vnu96LKcI30LOXsNqiN9gv4mYmge1lQrMXBqVVo8V7E/edit

We are scheduling this to go live by the end of the week as part of the Grafana Labs publication, and we are happy to have it published as part of the OTel publication as well, removing of course Grafana-specific parts (like the "we are hiring" part). 

Would it be acceptable to OTel's publication?

### PR DESCRIPTION
**Pull request by theletterf**
Fixes #5033.

### CODE REVIEW

File: content/en/blog/2024/otel-governance/index.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+title: Behind the scenes of the OpenTelemetry Governance Committee
+date: 2024-08-15
+author: '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (Grafana)'
+issue: 5033
+---
+
+![Chart showing what technologies are being used](header.png)
```
Comment (jpkrohling) [inline content/en/blog/2024/otel-governance/index.md:9]: Chart? Copy/paste issue? In any case, feel free to drop the header, as it has the Grafana logo in it, which makes it weird for the OTel publication.

File: content/en/blog/2024/otel-governance/index.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+---
+title: Behind the scenes of the OpenTelemetry Governance Committee
+date: 2024-08-15
+author: '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (Grafana)'
```
Comment (jpkrohling) [inline content/en/blog/2024/otel-governance/index.md:4]:
```suggestion
author: '[Juraci Paixão Kröhling](https://github.com/jpkrohling) (Grafana Labs)'
```

File: content/en/blog/2024/otel-governance/index.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+disappear is not an option. Mediation involves not only resolving disputes, but
+fostering a culture of open communication and mutual respect. By doing so, we
+ensure that our community remains a welcoming and productive environment for all
+contributors.
+
+## Reflections on the role
+
+Over the past three years, I’ve had the privilege of serving on the
+OpenTelemetry GC. I’m fortunate to work for a company that supports this role
+([we are hiring, by the way\!](https://grafana.com/about/careers/open-positions/)),
```
Comment (jpkrohling) [inline content/en/blog/2024/otel-governance/index.md:160]:
```suggestion
```

This line can go away, it's not appropriate for the OTel's publication. We'll have it on the Grafana's blog post.

**Review by jpkrohling — COMMENTED** [review]
Looks really good, thank you for working on this. Some vendor-specific parts need to be removed (the header and "we are hiring"), and the company name can be improved a bit, but other than that, LGTM.

**Review by jpkrohling — APPROVED** [review]

**Review by austinlparker — APPROVED** [review]

**Pull request merged**